client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -1752,7 +1752,7 @@ private StatusCode getPushDataFailCause(String message) {
     StatusCode cause;
     if (message == null) {
```

Comment:
Shall we also check `PUSH_DATA_FAIL_NON_CRITICAL_CAUSE_REPLICA` here?

Reply:
Thanks~, fixed